Java/PlatinumGenomes-variant-transformation/src/main/java/com/google/cloud/genomics/examples/TransformNonVariantSegmentData.java

```diff
@@ -153,8 +164,13 @@ private static TableSchema getTableSchema() {
     public void processElement(ProcessContext c) {
       Variant v = c.element();
 
+      HashMultiset genotypeCount = HashMultiset.create();  // This will typically hold counts for genotypes -1,0,1,2.
```

Comment:
Is this comment valuable here? I mostly ask because "typically" threw me with regards to what specifically it is modifying.

Are you just trying to add this comment:

// Typical genotypes observed are -1,0,1,2

Or is there something more you want to say (and so the comment should go above this line and be more verbose)?